cms/static/js/views/paging.js

```diff
@@ -0,0 +1,143 @@
+;(function (define) {
```

Comment:
I just saw your other comment that you had difficulty preserving history through moves. Let's take a look at this together in the morning, and if it isn't feasible I'll have to let it go.